Objective-C/Internal/CBLDatabase+Internal.h

```diff
@@ -77,6 +77,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable NSString*) getCookies: (NSURL*)url error: (NSError**)error;
 - (BOOL) saveCookie: (NSString*)cookie url: (NSURL*)url;
 
+- (CBLCollection*) defaultCollectionOrThrow;
```

Comment:
There is no need to declare it here unless there are somewhere else beside the database is using.